### PR TITLE
Create 2021-08-23-free-virtual-developer-day-auth-for-all.md

### PR DESCRIPTION
**Pull request by havi**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ x] A title that's approved by Dev Advocacy
- [ ] A URL that's approved by Dev Advocacy
- [ x] The content has been ran through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- ""
+- ""
+Image: blog/
+type: awareness|conversion
+---
+
+# Developer Day 2021: A Free Virtual Event to Celebrate the Future of Identity 
+
+
+It’s been a big year for Okta and Auth0. Our two companies have officially joined forces over the past few months, and we are excited about the chance to shape the future of identity together. It’s great to celebrate our combined developer communities at the first Developer Day event.
```
Comment (bdemers) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
---

It’s been a big year for Okta and Auth0. Our two companies have officially joined forces over the past few months, and we are excited about the chance to shape the future of identity together. It’s great to celebrate our combined developer communities at the first Developer Day event. 
```

The `H1`/ top level heading gets added automatically from the `title` field above.  Basically, everything in the post needs to be an `h2` or lower

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+by: advocate 
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Announcing Developer Day 2021: Auth for All with Okta and Auth0. Check out our speaker lineup, and register now. "
+tags: [identity, authorization, authentication, developer community]
+tweets:
+- ""
+- ""
+- ""
+Image: blog/
+type: awareness|conversion
```
Comment (bdemers) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
type: awareness
```

The template uses a pipe, but build will make you pick one or the other

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+layout: blog_post
+title: "Developer Day 2021:  Authentication, Authorization, and Celebrating Our Diverse Developer Community"
+Author: ali-spivak
+by: advocate 
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Announcing Developer Day 2021: Auth for All with Okta and Auth0. Check out our speaker lineup, and register now. "
+tags: [identity, authorization, authentication, developer community]
```
Comment (bdemers) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
tags: [identity, authorization, authentication, developer-community]
```

IIRC, tags cannot have spaces

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+[ Kapehe](https://www.sanity.io/exchange/community/kapehe): Kapehe is a talented Sanity.io developer who finds joy in helping devs create, whether it’s a quick JAMstack project or a reliable content backend. She’s been working closely with the developer community and sharing her knowledge through her blog, YouTube channel, conference talks, and podcasts.
+
+## Register to Participate in Developer Day
+
+If you haven’t already registered for Developer Day, please head to our [Developer Day website](https://www.okta.com/developerday/) to sign up – it’s free and virtual, so you can tune in from wherever you are. And you can join our free, interactive learning sessions on August 25th by visiting the [Developer Day Labs livestream](https://developer-day.live/).
+
+I hope to see you there!
+
+
+Note: Recordings of the Developer Day sessions will be available on the event website for several weeks after the event, so don’t hesitate to register even if you can’t join us synchronously in real-time.
```
Comment (bdemers) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
>**Note:** Recordings of the Developer Day sessions will be available on the event website for several weeks after the event, so don’t hesitate to register even if you can’t join us synchronously in real-time. 
```

Add **bold** marker on "note".  if you want to add the blue background, you can stick a `>` in front to format that block as a quote

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+tags: [identity, authorization, authentication, developer community]
+tweets:
+- ""
+- ""
+- ""
+Image: blog/
+type: awareness|conversion
+---
+
+# Developer Day 2021: A Free Virtual Event to Celebrate the Future of Identity
```
Comment (havi) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
# Developer Day 2021: Celebrating the Future of Identity
```
per mraible's excellent suggestion

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+[Jessica Temporal](https://jtemporal.com/): Jessica is a senior developer advocate at Auth0 and is the co-founder and co-host of Pizza de Dados, the first (and most beloved) Brazilian podcast about data science. She’s an active participant in PyLadies, an international network for promoting and empowering women in technology focused on the Python open-source community.
+
+[ Kapehe](https://www.sanity.io/exchange/community/kapehe): Kapehe is a talented Sanity.io developer who finds joy in helping devs create, whether it’s a quick JAMstack project or a reliable content backend. She’s been working closely with the developer community and sharing her knowledge through her blog, YouTube channel, conference talks, and podcasts.
+
+## Register to Participate in Developer Day
+
+If you haven’t already registered for Developer Day, please head to our [Developer Day website](https://www.okta.com/developerday/) to sign up – it’s free and virtual, so you can tune in from wherever you are. And you can join our free, interactive learning sessions on August 25th by visiting the [Developer Day Labs livestream](https://developer-day.live/).
```
Comment (havi) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
If you haven’t already registered, please head to our [Developer Day website](https://www.okta.com/developerday/) to sign up – it’s free and virtual, so you can tune in from wherever you are. And you can join our free, interactive learning sessions on August 25th by visiting the [Developer Day Labs livestream](https://developer-day.live/).
```
avoid repetition

**Review by havi — COMMENTED** [review]
thanks for reviewing so quickly. next I will attempt to add an image.

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
+---
+layout: blog_post
+title: "Developer Day 2021:  Authentication, Authorization, and Celebrating Our Diverse Developer Community"
+Author: ali-spivak
```
Comment (havi) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
Author: [ali-spivak]
```
syntax was missing

Comment (bdemers) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
author: ali-spivak
```

It's likely a case-sensitive issue and not the brackets (the brackets are only needed if you have more than one, but it shouldn't hurt either)

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+author: ali-spivak
+by: advocate 
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Announcing Developer Day 2021: Auth for All with Okta and Auth0. Check out our speaker lineup, and register now. "
+tags: [identity, authorization, authentication, developer-community, conference]
+tweets:
+- ""
+- ""
+- ""
+Image: blog/free-virtual-developer-day-auth-for-all/social.png
```
Comment (mraible) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]: You can also delete the four `tweets` lines if you're not going to use them. 
```suggestion
image: blog/free-virtual-developer-day-auth-for-all/social.png
```

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+layout: blog_post
+title: "Developer Day 2021:  Authentication, Authorization, and Celebrating Our Diverse Developer Community"
```
Comment (havi) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
title: "Developer Day 2021: Celebrating the Future of Identity"
```
Updating to the current title

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+tags: [identity, authorization, authentication, developer-community, conference]
+tweets:
+- ""
+- ""
+- ""
+Image: blog/free-virtual-developer-day-auth-for-all/social.png
+type: awareness
+---
+
+# Developer Day 2021: Celebrating the Future of Identity
```
Comment (havi) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion

```
remove H1 copy

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+
+It's been a big year for Okta and Auth0. Our two companies have officially joined forces over the past few months, and we are excited about the chance to shape the future of identity together. It's great to celebrate our combined developer communities at the first Developer Day event. 
+
+I look forward to exploring more ways to bring our communities together in the future in collaboration with our Auth0 counterparts.
```
Comment (havi) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
I look forward to exploring more ways we can collaborate with our Auth0 counterparts and far-flung global communities through future events and activities. 
```
avoiding redundant language

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
+---
+layout: blog_post
+title: "Developer Day 2021:  Authentication, Authorization, and Celebrating Our Diverse Developer Community"
+author: ali-spivak
+by: advocate 
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Announcing Developer Day 2021: Auth for All with Okta and Auth0. Check out our speaker lineup, and register now. "
+tags: [identity, authorization, authentication, developer-community, conference]
+tweets:
```
Comment (havi) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md:9]:
```suggestion
tweets:
"It's free, it's virtual, and you can still sign up for Developer Day on August 24th! Tune in from wherever you are to explore the future of identity. Join our free, interactive learning sessions on August 25th. "
```
one new tweet

**Review by havi — COMMENTED** [review]
one more round of teensy changes & corrections

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+"It's free, it's virtual, and you can still sign up for Developer Day on August 24th! Tune in from wherever you are to explore the future of identity. Join our free, interactive learning sessions on August 25th. "
+- ""
+- ""
+- ""
+image: blog/free-virtual-developer-day-auth-for-all/social.png
```
Comment (bdemers) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
"It's free, it's virtual, and you can still sign up for Developer Day on August 24th! Tune in from wherever you are to explore the future of identity. Join our free, interactive learning sessions on August 25th. "
image: blog/free-virtual-developer-day-auth-for-all/social.png
```

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+type: awareness
+---
+
+
+
+It's been a big year for Okta and Auth0. Our two companies have officially joined forces over the past few months, and we are excited about the chance to shape the future of identity together. It's great to celebrate our combined developer communities at the first Developer Day event.
```
Comment (bdemers) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
type: awareness
---

It's been a big year for Okta and Auth0. Our two companies have officially joined forces over the past few months, and we are excited about the chance to shape the future of identity together. It's great to celebrate our combined developer communities at the first Developer Day event. 
```

File: _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md [hunk 15]
```diff
@@ -0,0 +1,45 @@
+---
+layout: blog_post
+title: "Developer Day 2021: Celebrating the Future of Identity"
+author: ali-spivak
+by: advocate 
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Announcing Developer Day 2021: Auth for All with Okta and Auth0. Check out our speaker lineup, and register now. "
+tags: [identity, authorization, authentication, developer-community, conference]
+tweets:
+"It's free, it's virtual, and you can still sign up for Developer Day on August 24th! Tune in from wherever you are to explore the future of identity. Join our free, interactive learning sessions on August 25th. "
```
Comment (bdemers) [inline _source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md]:
```suggestion
tweets:
- "It's free, it's virtual, and you can still sign up for Developer Day on August 24th! Tune in from wherever you are to explore the future of identity. Join our free, interactive learning sessions on August 25th. "
```

**Review by bdemers — APPROVED** [review]

**Pull request merged**